### PR TITLE
Authentication

### PR DESCRIPTION
**Pull request by snayerman**

### CODE REVIEW

File: src/Users/Login.js [hunk 1]
```diff
@@ -17,35 +20,70 @@ export default class Login extends Component {
...
          <>
+            {this.state.modal ? this.renderModal() : null}
+
             <div>LOGIN</div>
             <div className="login-body">
                <form className="form-container" noValidate autoComplete="off">
                   <div className="text-fields">
                      <TextField
-                        id="outlined-name"
                         label="User Name"
```
Comment (kramsey458) [inline src/Users/Login.js]: Change label to email

File: src/Users/Login.js [hunk 2]
```diff
@@ -17,35 +20,70 @@ export default class Login extends Component {
...
     };
 
+   authenticate = () => {
+      let url = 'https://capstone-plumbers-api.herokuapp.com/users/auth';
+
+      Axios.post(url, {
+         email: this.state.email,
+         password: this.state.password
+      }).then(res => {
+         console.log(res);
```
Comment (tonyc856) [inline src/Users/Login.js]: Just a reminder to remove these console logs if you haven't already :)

File: src/Users/Login.js [hunk 3]
```diff
@@ -17,35 +20,70 @@ export default class Login extends Component {
...
+         window.localStorage.setItem('loggedInUser', JSON.stringify({
+            email: res.data.data.email,
+            token: res.data.data.token
+         }));
+         this.openModal(1, "", () => {
+            this.props.history.push('/');
+         });
+      }).catch(err => {
+         console.log(err.response);
```
Comment (tonyc856) [inline src/Users/Login.js]: Remove the one here as well

File: src/Users/Signup.js [hunk 4]
```diff
@@ -17,44 +21,75 @@ export default class Signup extends Component {
...
     };
 
+   register = () => {
+      let url = 'https://capstone-plumbers-api.herokuapp.com/users/signup';
+
+      Axios.post(url, {
+         email: this.state.email,
+         password: this.state.password
+      }).then(res => {
+         console.log(res);
```
Comment (tonyc856) [inline src/Users/Signup.js]: Here as well

File: src/Users/Signup.js [hunk 5]
```diff
@@ -17,44 +21,75 @@ export default class Signup extends Component {
...
+         this.openModal(1, "", () => {
+            this.props.history.push('/login');
+         });
+      }).catch(err => {
+         console.log(err.response);
```
Comment (tonyc856) [inline src/Users/Signup.js]: And here

File: src/Users/Login.js (hunk 3 above)
Comment (snayerman) [reply src/Users/Login.js]: I don't think we should remove comments from the error handler. We need to be able to see what went wrong if there's an error.

**Comment by snayerman** [comment]
Fixed changes :)))

**Pull request merged**